CONTRIBUTING.md

```diff
@@ -126,26 +126,18 @@ This allows people to (1) use PyBaMM without importing optional dependencies by
 
 **Writing Tests for Optional Dependencies**
 
-Whenever a new optional dependency is added for optional functionality, it is recommended to write a corresponding unit test in `test_util.py`. This ensures that an error is raised upon the absence of said dependency. Here's an example:
+Below, we list the currently available test functions to provide an overview. If you find it useful to add new test cases please do so within `tests/unit/test_util.py`.
 
-```python
-from tests import TestCase
-import pybamm
-
-
-class TestUtil(TestCase):
-    def test_optional_dependency(self):
-        # Test that an error is raised when pybtex is not available
-        with self.assertRaisesRegex(
-            ModuleNotFoundError, "Optional dependency pybtex is not available"
-        ):
-            sys.modules["pybtex"] = None
-            pybamm.function_using_pybtex(x, y, z)
-
-        # Test that the function works when pybtex is available
-        sys.modules["pybtex"] = pybamm.util.import_optional_dependency("pybtex")
-        pybamm.function_using_pybtex(x, y, z)
-```
+Currently, there are three functions to test what concerns optional dependencies:
+- `test_import_optional_dependency`
```

Comment:
It's be nice to link the valid URLs of these functions so that one can be directed to the code.

Reply:
The URLs for the tests can change and their permalinks for the relevant can also get outdated (in case we move the tests to somewhere else, rewrite the tests later, etc.), do we have a better way to link these?